setup.py

```diff
@@ -16,6 +16,24 @@
     exec(f.read(), version_globals, version_locals)
     streaming_version = version_locals['__version__']
 
+# Use repo README for PyPi description
+with open('README.md', 'r', encoding='utf-8') as fh:
```

Comment:
i am curious why we need to specify r and utf-8 here?

Reply:
- `r`: we don't want to modify readme. The change that we are making is for pypi website.
- `utf-8`: UTF-8 is one of the most commonly used encodings, and Python often defaults to using it.